### DISCUSSION TITLE
Cross-compatibility with simai

### DISCUSSION THREAD

**Discussion by TwoQuantumBits** [LumpBloom7/sentakki#518] [Ideas]
As of right now, the other popular maimai map/chart player is AstroDX, which [parses maps based on the simai format](https://github.com/reflektone-games/SimaiSharp). Because of that, a few mappers/charters are pretty used to working with simai. As such, I wonder if we can have cross-compatibility with simai in the future (i.e. simai charts can be imported as custom sentakki maps and vice versa)?

**Comment by LumpBloom7** [comment]
Yes. I think a program to convert simai format charts to osu/sentakki format (and vice versa) will be available in the future. Currently the osu beatmap format doesn't support newer hitobject types, so that will need to be a thing before any conversions can be done, it's also why the sentakki editor is not available for public use yet (since there aren't any formats to save to).

Relevant osu side issue: https://github.com/ppy/osu/issues/10314
